controllers/storagecluster/storageclasses.go

```diff
@@ -452,16 +438,7 @@ func newCephOBCStorageClassConfiguration(initData *ocsv1.StorageCluster) Storage
 // newStorageClassConfigurations returns the StorageClassConfiguration instances that should be created
 // on first run.
 func (r *StorageClusterReconciler) newStorageClassConfigurations(initData *ocsv1.StorageCluster) ([]StorageClassConfiguration, error) {
-	ret := []StorageClassConfiguration{
-		newCephFilesystemStorageClassConfiguration(initData),
-		newCephBlockPoolStorageClassConfiguration(initData),
-	}
-
-	//TODO: will be removed when we start handling kubevirt class through provider-client communication
-	// If kubevirt crd is present, we create a specialized rbd storageclass for virtualization environment
-	if r.AvailableCrds[VirtualMachineCrdName] {
-		ret = append(ret, newCephBlockPoolVirtualizationStorageClassConfiguration(initData))
-	}
+	ret := []StorageClassConfiguration{}
 
 	if initData.Spec.ManagedResources.CephNonResilientPools.Enable {
```

Comment:
i though we will move the replica 1 and later NFS to client as well. isnt it?

Reply:
NFS will be handled in a different PR there is a different story to track it [RHSTOR-6922](https://issues.redhat.com/browse/RHSTOR-6922), I will remove the replica 1 storageClass from here